### ISSUE TITLE
loss can't be reduced during the recog training

### ISSUE THREAD

**Issue by speaknowpotato** [open-mmlab/mmocr#827]
hi team, 
it's my first time to run some big dataset training on mmocr. 
I find my loss is stuck between 2 and 3 and can't be reduced anymore. 
I am using **sar** as the model and downloading the **IIIT5K** dataset as the training dataset. 
here is the log file. 
Any help is appreciated. thanks!

[20220310_224916.log](https://github.com/open-mmlab/mmocr/files/8230033/20220310_224916.log)

**Comment by xinke-wang** [comment]
Hi,

From the log file, I noticed that the `total_epochs` is set to `5`, the recognition model is unable to converge in such a short schedule without pre-training on the synthetic dataset. My recommendation is that you could try to set a longer training schedule or pre-train the model with SynthText.

**Comment by speaknowpotato** [comment]
thanks so much @xinke-wang . 
I will try both methods to see how it works. 
for option 1, I will set the epoch to 50
for option 2, do you mean I should train the model with SynthText first? and retrain it with IIIT5K?

**Comment by xinke-wang** [comment]
> thanks so much @xinke-wang . I will try both methods to see how it works. for option 1, I will set the epoch to 50 for option 2, do you mean I should train the model with SynthText first? and retrain it with IIIT5K?

Hi @speaknowpotato ,

1. I'm not sure if 50 epochs are enough, but you can have a try at first. I also recommend you to refer to our official SAR pre-trained model [here](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/sar/README.md). We use 8 datasets to train the SAR model together for 5 epochs, while the IIIT5K `repeat_num` was set to 20, which equals `5 x 20 = 100 epochs`.

2. Yes, usually, we do not train a recognizer from scratch, but pretrain the model with synthetically generated texts. The mainstream strategy is whether to train the model with a combination of real and synthetic data or firstly train with synthetic data then fine-tune with real data. You can see [here](https://github.com/open-mmlab/mmocr/blob/main/configs/textrecog/sar/README.md) that we used a combination of synthetic data, including `SynthText`, `SynthAdd`, and `Syn90k`, and real data, including `IIIT5K` and etc.

**Comment by gaotongxiao** [comment]
I'm closing this issue, but feel free to reopen it if you have any followup questions.
